static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js

```diff
@@ -3,6 +3,7 @@ define([
     'common/utils/location',
```

Comment:
We should ideally marry any changes in this file with some tests in commercial-feature-policies.spec.js, too.